p/api/fever.php

```diff
@@ -0,0 +1,638 @@
+<?php
+/**
+ * Fever API for FreshRSS
+ * Version 0.1
+ * Author: Kevin Papst / https://github.com/kevinpapst
+ *
+ * Inspired by:
+ * TinyTinyRSS Fever API plugin @dasmurphy
+ * See https://github.com/dasmurphy/tinytinyrss-fever-plugin
+ */
+
+// ================================================================================================
+// BOOTSTRAP FreshRSS
+require(__DIR__ . '/../../constants.php');
+require(LIB_PATH . '/lib_rss.php');    //Includes class autoloader
+Minz_Configuration::register('system', DATA_PATH . '/config.php', FRESHRSS_PATH . '/config.default.php');
+
+// check if API is enabled globally
+FreshRSS_Context::$system_conf = Minz_Configuration::get('system');
+if (!FreshRSS_Context::$system_conf->api_enabled) {
+	Minz_Log::warning('serviceUnavailable() ' . debugInfo(), API_LOG);
+	header('HTTP/1.1 503 Service Unavailable');
+	header('Content-Type: text/plain; charset=UTF-8');
+	die('Service Unavailable!');
+}
+
+ini_set('session.use_cookies', '0');
+register_shutdown_function('session_destroy');
+Minz_Session::init('FreshRSS');
+// ================================================================================================
+
+
+class FeverAPI_EntryDAO extends FreshRSS_EntryDAO
+{
+	/**
+	 * @return array
+	 */
+	public function countFever()
+	{
+		$values = array(
+			'total' => 0,
+			'min' => 0,
+			'max' => 0,
+		);
+		$sql = 'SELECT COUNT(id) as `total`, MIN(id) as `min`, MAX(id) as `max` FROM `' . $this->prefix . 'entry`';
+		$stm = $this->bd->prepare($sql);
+		$stm->execute();
+		$result = $stm->fetchAll(PDO::FETCH_ASSOC);
+
+		if (!empty($result[0])) {
+			$values = $result[0];
+		}
+
+		return $values;
+	}
+
+	/**
+	 * @param string $prefix
+	 * @param array $values
+	 * @param array $bindArray
+	 * @return string
+	 */
+	protected function bindParamArray($prefix, $values, &$bindArray)
+	{
+		$str = "";
+		foreach ($values as $index => $value) {
+			$str .= ":".$prefix.$index.",";
```

Comment:
Could you please cast `$index` to int (or make a normal `for` loop)? I am worried there might be ways of arriving here with an associative array in `$values` and a risk of SQL injection.